app.js

```diff
@@ -22,4 +26,18 @@ app.get('/', (req, res) => {
   res.render('index');
 });
 
+app.get('/beers',(req,res) => {
+  punkAPI
+  .getBeers()
+  .then(beersFromApi => res.render('beers', {beers : beersFromApi}))
+  .catch(error=>console.log(error))
+})
+
+app.get('/random-beers',(req,res)=>{
+  punkAPI
+  .getRandom()
+  .then(responseFromApi => res.render('random-beers', {ranBeer : responseFromApi}))
```

Comment:
You correctly handled ranBeer as an array in the hbs. Another way to do it, is to just send the element to render. Since it is an array of one element and will always be.

`.then(responseFromApi => res.render('random-beers', {ranBeer : responseFromApi[0] }))`